torchmultimodal/modules/layers/normalizations.py

```diff
@@ -72,3 +72,25 @@ def _norm(self, x: Tensor) -> Tensor:
     def forward(self, x: Tensor) -> Tensor:
         x_normed = self._norm(x.float()).type_as(x)
         return x_normed * self.scale
+
+
+class SimpleRMSNorm(nn.Module):
+    """Simple RMSNorm
+
+    SRMSNorm(x) = (x / ∥x∥2) /√d
```

Comment:
Should be this, right?
```suggestion
    SRMSNorm(x) = x / (∥x∥2 /√d)
```